internal/v1/api.yaml

```
@@ -606,6 +606,35 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/HTTPErrorList'
+  /experimental/blueprints/{id}/detail:
```

According to REST best practices, this can be as simple as:

```suggestion
  /experimental/blueprints/{id}:
```

If there is precedens, let’s keep that.